js/admin.js

```diff
@@ -950,14 +950,16 @@ function Admin() {
                     data: data,
 
                     success: function (data, textStatus) {
+                        data = data.data;
                         $(".loader-image").hide();
 
                         if (data.status == "error") {
                             system.addAlert("error", data.errorText);
                             return;
                         }
 
-                        location.reload();
+                        document.location.href="/project/" + data.project
```

Comment:
Hardcoding URLs is not allowed - we may change them in future. Find some other way to pass the URL here - for example pass it in response.